spec/controllers/v1/people_controller_spec.rb

```diff
@@ -74,9 +76,9 @@
 
         it "updates location" do
           expect_any_instance_of(Location).to receive(:update_location).
-            with(address: '2020 Oregon St', city: nil, state: nil, zip: '94703') { true }
+            with(address: '2020 Oregon St', zip: '94703') { true }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.